src/main/resources/db/migration/2001224.sql

```diff
@@ -0,0 +1,3 @@
+# 2.1.224
+
+ALTER TABLE StoryBookParagraph DROP COLUMN audio;
```

Comment:
@watr-sheep The column name needs to be adjusted:

```
> DESC StoryBookParagraph;
+---------------------+---------------+------+-----+---------+----------------+
| Field               | Type          | Null | Key | Default | Extra          |
+---------------------+---------------+------+-----+---------+----------------+
| id                  | bigint(20)    | NO   | PRI | NULL    | auto_increment |
| originalText        | varchar(1024) | NO   |     | NULL    |                |
| sortOrder           | int(11)       | NO   |     | NULL    |                |
| storyBookChapter_id | bigint(20)    | YES  | MUL | NULL    |                |
| audio_id            | bigint(20)    | YES  | MUL | NULL    |                |
+---------------------+---------------+------+-----+---------+----------------+
```

Reply:
Fixed it.